### PR TITLE
Add AdvancedAuditing e2e test jobs for master and 1.7

### PR DESCRIPTION
**Pull request by timstclair**
Workaround for https://github.com/kubernetes/kubernetes/issues/47283

/cc @dchen1107

### CODE REVIEW

**Comment by dchen1107** [comment]
@k8s-bot pull-test-infra-bazel test this

File: jobs/config.json [hunk 1]
```diff
@@ -491,6 +491,31 @@
...
+  "ci-kubernetes-e2e-gce-audit": {
+    "args": [
+      "--env-file=jobs/platform/gce.env", 
+      "--env-file=jobs/ci-kubernetes-e2e-gce-audit.env", 
+      "--timeout=180m", 
+      "--extract=ci/latest"
+    ], 
+    "scenario": "kubernetes_e2e", 
+    "sigOwners": [
+      "UNKNOWN"
```
Comment (dchen1107) [inline jobs/config.json]: why UNNOWN, instead of sig/auth?

File: jobs/config.json [hunk 2]
```diff
@@ -491,6 +491,31 @@
...
+    "args": [
+      "--env-file=jobs/platform/gce.env", 
+      "--env-file=jobs/ci-kubernetes-e2e-gce-audit-release-1-7.env", 
+      "--extract=ci/latest-1.7", 
+      "--timeout=180m", 
+      "--mode=local"
+    ], 
+    "scenario": "kubernetes_e2e", 
+    "sigOwners": [
+      "UNKNOWN"
```
Comment (dchen1107) [inline jobs/config.json]: ditto.

File: jobs/config.json [hunk 3]
```diff
@@ -1964,6 +1989,31 @@
...
+    "args": [
+      "--env-file=jobs/platform/gce.env", 
+      "--env-file=jobs/ci-kubernetes-e2e-gci-gce-audit-release-1-7.env", 
+      "--extract=ci/latest-1.7", 
+      "--timeout=180m", 
+      "--mode=local"
+    ], 
+    "scenario": "kubernetes_e2e", 
+    "sigOwners": [
+      "UNKNOWN"
```
Comment (dchen1107) [inline jobs/config.json]: ditto.

**Comment by dchen1107** [comment]
Travis CI is failing.

File: prow/config.yaml [hunk 4]
```diff
@@ -5930,6 +5930,72 @@ periodics:
...
+    volumes:
+    - name: service
+      secret:
+        secretName: service-account
+    - name: ssh
+      secret:
+        defaultMode: 256
+        secretName: ssh-key-secret
+
+- interval: 1h
```
Comment (krzyzacy) [inline prow/config.yaml]: please add the master prow jobs here as well

Comment (krzyzacy) [inline prow/config.yaml]: I'll add a commit to your PR

**Pull request closed without merging**